ros_ws/src/autonomous/evolutionary/scripts/train.py

```diff
@@ -18,24 +22,34 @@ def __init__(self):
         rospy.Subscriber(TOPIC_CRASH, Empty, self.on_crash)
 
         self.population = []
-        self.untested_population = [NeuralCarDriver()
-                                    for _ in range(POPULATION_SIZE)]
-        self.current_driver = self.untested_population[0]
 
         self.is_terminal_step = False
         self.episode_length = 0
         self.generation = 0
         self.test = 0
 
+        self.untested_population = []
+        self.spawn_forward = True
+        if CONTINUE_TRAINING:
+            for i in range(POPULATION_SIZE):
+                individuum = NeuralCarDriver()
```

Comment:
This should be renamed to an english word, such as "driver", "agent" or "network"